### ISSUE TITLE
While building Ragnar I get an error

### ISSUE THREAD

**Issue by Zalarv** [cococry/Ragnar#54]
Using the install script, pkgbuild, or building it on my own I get this error that looks like this:

cc -c -03 -ffast-math -Wall -Wextra -pedantic -lxcb -lxcb-keysms -lxcb-icccm -lxcb-cursor -lxcb-randr -lxb-composit -lxcb-ewmh -lX11 -lX11-xcb -IGL -lleifeif -lclipboard -lm ragnar.c
ragnar.c :13:10: fatal error: xcb/xcb_cursor.h: No such file or directory
   13 | #include <xcb/xcb_cursor.h>
         |
compilation terminated.
make: *** [Makefile:20: Ragnar.o] Error 1
==> ERROR: A failure occured in build().
       Aborting..
->error making: ragnarwm- exit status 4
-> Failed to install the following packages. Manual intervention required:
ragnarwm - exit status 4

Additional context:
I have tried to build this on void linux, arch, and artix and have used a tty so I can't provide a screenshot 

Also there might be some spelling errors for reasons listed above

**Comment by PUXSY** [comment]
me to what to do? 
do u have a fix??

**Comment by Zalarv** [comment]
I do not understand what you said there.

**Comment by PUXSY** [comment]
I have the same problem, did you find the solution?

**Comment by Zalarv** [comment]
No, I have moved on to river. The dev seems to be on break or has moved on from the project entirely.

**Comment by PUXSY** [comment]
Ahhh ok thanks

**Comment by PUXSY** [comment]
river works on ubuntu?

**Comment by Zalarv** [comment]
Though, do look out because there could be a rewrite in the future.

**Comment by PUXSY** [comment]
yes of course

**Comment by PUXSY** [comment]
@Zalarv  river works on ubuntu?

**Comment by Zalarv** [comment]
Probably. I haven't tried ubuntu myself. Just look a the repo for build instructions and you'll most likely be fine

**Comment by kingdomkind** [comment]
Probably missing a dependency

**Comment by cococry** [comment]
Install the decadences stated in the README, they are needed for the build to work.

You don’t have xcb_cursor installed, but maybe also more depdencies, use your distros Package Manager to install them and the build should run fine.
